`content` rules from the JS config that are also covered by the automatic source detection should not be migrated to CSS (#14714)
tailwindlabs/tailwindcss: CHANGELOG.md

## [Unreleased]

### Changed

> - _Upgrade (experimental)_: `content` rules from the JS configuration that are also covered by the automatic source detection are no longer migrated to CSS

Reviewer: ```suggestion
- _Upgrade (experimental)_: Don't create `@source` rules for `content` paths that are already covered by automatic source detection
```
